packages/orval/package.json

```diff
@@ -33,14 +37,6 @@
     "zod",
     "hono"
   ],
-  "author": {
-    "name": "Victor Bury",
-    "email": "victor@anymaniax.com"
-  },
```

Comment:
![ed2](https://github.com/user-attachments/assets/a815c3c1-1b75-49d5-bf1a-4fefb87757fd)

Reply:
Should I retain the author details?

Reply:
Should be fine as is